public/data/manishjha-04.json

```
@@ -0,0 +1,12 @@
+{
+  "name": "Manish Jha",
+  "location": "India",
+  "bio": "A Mern Enthusisast , Frontend Virtuoso and  Blockchain Visionary",
+  "avatar": "https://github.com/manishjha-04.png",
+  "portfolio": "https://github.com/manishjha-04",
+  "skills": ["Your Skill 1", "Your Skill 2", "..."],
```

```suggestion
  "skills": ["Your Skill 1", "Your Skill 2", "..."],
```
Add your skills here @manishjha-04

---

Done Sir